src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -319,7 +319,7 @@ private void UpdateReferenceLocationsTask()
             // https://github.com/dotnet/roslyn/issues/40890
             await _threadingContext.JoinableTaskFactory.SwitchToMainThreadAsync(alwaysYield: true, cancellationToken);
 
-            RaiseSessionSpansUpdated(inlineRenameLocations.Locations.ToImmutableArray());
+            RaiseSessionSpansUpdated([.. inlineRenameLocations.Locations]);
```

Comment:
>Locations [](http://example.com/codeflow?start=63&length=9)

Another one that is interesting

Reply:
This becomes:

```c#
		int num = 0;
		InlineRenameLocation[] array = new InlineRenameLocation[locations.Count];
		foreach (InlineRenameLocation item in locations)
		{
			array[num] = item;
			num++;
		}
		RaiseSessionSpansUpdated(ImmutableCollectionsMarshal.AsImmutableArray(array));
```
which is again what we'd want here.